### PR TITLE
[minor] Improve readability + formatting

### PR DESCRIPTION
**Pull request by atb00ker**
No functional change, it only seems easier to read this way. Merge if you feel this provides readability improvement.

Blocked: #135

### CODE REVIEW

File: openwisp_radius/tests/test_social.py [hunk 1]
```diff
@@ -66,8 +66,8 @@ def test_redirect_cp_301(self):
             u.username, token.key, rad_token.key
         )
         self.assertIn(querystring, r.url)
-        u = User.objects.filter(username='socialuser').first()
-        self.assertIn((self.default_org.pk,), u.organizations_pk)
+        user = User.objects.filter(username='socialuser').first()
+        self.assertTrue(user.organizations_dict.get(str(self.default_org.pk), False))
```
Comment (atb00ker) [inline openwisp_radius/tests/test_social.py:70]: Unrelated, change because `organizations_pk` is deprecated.

**Comment by atb00ker** [comment]
Test will pass after https://github.com/openwisp/openwisp-radius/pull/135 is merged and rebased to this.

File: openwisp_radius/tests/test_social.py (hunk 1 above)
Comment (nemesisdesign) [reply openwisp_radius/tests/test_social.py:70]: why not:
```python
self.assertTrue(user.is_member(self.default_org))
self.assertFalse(user.is_manager(self.default_org))
```

Comment (nemesisdesign) [reply openwisp_radius/tests/test_social.py:70]: why not:
```python
self.assertTrue(user.is_member(self.default_org))
self.assertFalse(user.is_manager(self.default_org))
```

**Pull request merged**